cmd/clusterctl/client/cluster/mover.go

```diff
@@ -342,6 +343,17 @@ func (o *objectMover) move(graph *objectGraph, toProxy Proxy) error {
 	// - then all the MachineSets, then all the Machines, etc.
 	moveSequence := getMoveSequence(graph)
 
+	// Check if there are any nodes that were skipped from the move sequence and log them.
+	// This happens if the node has owners that will not be moved.
+	if len(moveSequence.skippedNodes) > 0 {
+		nodeNames := []string{}
+		for _, n := range moveSequence.skippedNodes {
+			nodeNames = append(nodeNames, fmt.Sprintf(" * %s", n.identityStr()))
+		}
+		warningMsg := fmt.Sprintf("[Warning] The following objects will not be moved because they are owned by objects that are not part of the move sequence: \n%s", strings.Join(nodeNames, "\n"))
```

Comment:
```suggestion
		warningMsg := fmt.Sprintf("[Warning] The following objects will not be moved because they are not part of the OwnerReferences chain considered by move or have an owner not included in it: \n%s", strings.Join(nodeNames, "\n"))
```

Reply:
Question: Should we consider also showing, for each of the skipped objects, which owner(s) is not part of the move sequence. Something like "object X is not moved because it is owned by Y that is not part of the move sequence"? 

We will need to do some ownerRef look ups to figure the actual names of the owners but do you think it is valuable?

Also, I am fine if we want to keep the message simple for this first iteration and improve on it later.